src/markdown-pages/docs/managers/cics-ts-manager.md

```diff
@@ -3,30 +3,66 @@ path: "/docs/managers/cics-ts-manager"
 title: "CICS TS Manager"
 ---
 
-**Alpha - This Manager is being actively developed. It is subject to change and has not been extensively tested.**
+This Manager is at Alpha level. You can view the <a href="https://javadoc.galasa.dev/dev/galasa/cicsts/package-summary.html" target="_blank" rel="noopener noreferrer">Javadoc documentation for the Manager here</a>.<br>
 
-## Overview
-This manager provides Galasa tests to CICS/TS functions.<br><br>
 
+[Overview](#overview)<br>
+[Configuring](#configuring)<br>
+[Provided annotation](#annotations)<br>
+[Code snippets and examples](#codesnippets)<br>
 
 
+# <a name="overview"></a>Overview
 
+This Manager provides Galasa tests to CICS/TS functions. 
 
-## Configuration Properties
 
-The following are properties used to configure the CICS TS Manager.
+
+
+## <a name="configuring"></a>Configuration Properties
+
+The following are properties that are used to configure the CICS TS Manager in the CPS.
+
+To provision a CICS Region for your test, as a minimum you need to configure the following properties for the CICS TS Manager:
```

Comment:
I'd perhaps say here instead, 'To connect your Galasa test to a developer supplied environment with a provisioned CICS Region' as their CICS Region should be provisioned and up and running already, and we are just connecting with these properties.

Reply:
updated